docker/Dockerfile

```diff
@@ -6,7 +8,7 @@ ARG TARGETOS
 ARG LDFLAGS
 RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -ldflags "${LDFLAGS}" -o _output/secrets-store-csi ./cmd/secrets-store-csi-driver
```

Comment:
If the Dockerfile container RUN, then I think we cannot use buildx to build windows container image on a linux machine. The workaround is to remote go build outside as a separate step.

Reply:
oh, I see this is only for Linux. just ignore my comment.